lib/gds_api/test_helpers/content_store.rb

```diff
@@ -9,10 +9,26 @@ module ContentStore
 
       CONTENT_STORE_ENDPOINT = Plek.current.find('content-store')
 
-      def content_store_has_item(base_path, body = content_item_for_base_path(base_path), expires_in = 900)
+      # Stubs a content item in the content store.
+      # The following options can be passed in:
+      #
+      #   :max_age  will set the max-age of the Cache-Control header in the response. Defaults to 900
+      #   :private  if true, the Cache-Control header will include the "private" directive. By default it
+      #             will include "public"
+      def content_store_has_item(base_path, body = content_item_for_base_path(base_path), options = {})
```

Comment:
This is a breaking change, so technically should require a major version bump.  (Good timing because they's already another breaking change lined up - #322).